src/components/charts/galacticProperties/index.jsx

```diff
@@ -63,12 +63,43 @@ class GalacticProperties extends React.Component {
     if (isNewActiveGalaxy) {
       this.updateSelectedData(activeGalaxy);
     }
+
+    if (updateDomain) {
+      this.updateDomain();
+    }
   }
 
   componentWillUnmount() {
     this.removeEventListeners();
   }
 
+  updateDomain() {
+    const {
+      height,
+      width,
+      padding,
+      domain,
+      xDomain,
+      yDomain,
+      domainUpdatedCallback,
+    } = this.props;
+
+    this.setState(
+      prevState => ({
+        ...prevState,
+        xScale: d3ScaleLinear()
+          .domain(domain ? domain[0] : xDomain)
+          .range([padding, width]),
+        yScale: d3ScaleLinear()
+          .domain(domain ? domain[1] : yDomain)
+          .range([height - padding, 0]),
```

Comment:
DRY. You got the same thing here as above in the constructor.  break it our into one or more methods like getXScale getYScale and then reuse in both spots (see LightCurve component for an example of this)